client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -379,9 +405,14 @@ export class CheckoutThankYou extends React.Component {
 			if ( delayedTransferPurchase ) {
 				planProps = {
 					action: (
-						<a className="thank-you-card__button" onClick={ this.startTransfer }>
+						<button
+							className="checkout-thank-you__button"
+							onClick={ this.startTransfer }
+							tabIndex={ 0 }
+							onKeyPress={ () => {} }
```

Comment:
Should this also start the transfer?

Reply:
I rolled back the attempt to update the for linting. I can circle back once we get this merged